include/oneapi/dpl/pstl/utils.h

```diff
@@ -128,6 +128,21 @@ class __pstl_assign
     }
 };
 
+template <typename _Comp, typename _Proj>
+class __compare
+{
+    _Comp __comp;
+    _Proj __proj;
```

Comment:
I believe you need a constructor with this as a class and these as private member variables.

Reply:
renaned to struct